src/App.test.tsx

```diff
@@ -268,14 +309,14 @@ describe('App component', () => {
 
     it('displays the quiz progress', async () => {
       const user = userEvent.setup();
-      render(<App data={testDataWithFourQuestions}></App>, {
+      const testDataWith2Questions = data.slice(0, 2);
+      render(<App data={testDataWith2Questions}></App>, {
         wrapper: translationsProvider,
       });
 
-      const button = screen.getByText(/Medium set/i);
+      const button = screen.getByText(/Complete set/i);
       await user.click(button);
 
-      // A medium set of a test with four questions contains 2 questions
       let indexTextEl = screen.getByText('1 of 2');
 
       for (let i = 0; i < 5; i++) {
```

Comment:
Why do we click here 5 times in the loop when the set only has 2 questions now? It should be sufficient to only click 3 times to see the progress updates.

Reply:
No, because questions appear alternately. In the first loop you answer question 1 wrong, in the second loop you answer question 2 wrong and it won't be until loop 5 that question 1 expires the attempts. If we were to continue, in loop 6 the attempts for question 2 would expire and therefore the quiz would end.